src/olympia/files/tests/test_views.py

```diff
@@ -457,13 +458,9 @@ def test_unicode(self):
         res = self.client.get(self.file_url(u'\u1109\u1161\u11a9'))
```

Comment:
This test was fixed by changing the zip file to set the relevant encoding flag in it, because Python 3 behavior for zipfiles without that flag is different now.

Before, it used to return the filenames as bytes, and we'd decode that using utf-8 ourselves. Now, *if the flag is absent* it uses `cp437`, which the the "legacy" encoding for zipfiles, and otherwise falls back to utf-8. The end result for zip files without the flag set is the filenames may look weird in some circumstances, but ultimately should still be coherent.

Reply:
Especially since we don't really touch or modify or extract them anymore. I think that should work, might be worth getting QA trying to break it though.